bip-emilengler-replacing-the-term-address.mediawiki

```diff
@@ -0,0 +1,46 @@
+<pre>
+  BIP: UNDEFINED
+  Titel: Replacing the term address
```

Comment:
Suggest "Terminology for payment recipient identifiers" or something that refers to the new standard rather than the previous standard.

Reply:
I think to get more attention it would be better to include the old de-facto standard into the title

Reply:
I don't agree. The title is supposed to be for the BIP, not an older thing being replaced.

Reply:
Convinced, updated.
This should be the final title version of the BIP title however.
Sorry for letting you update it again :confused: